charts/testkube-enterprise/installer_enterprise.sh

```diff
@@ -130,15 +130,55 @@ request_license() {
     parse_license_key
 }
 
+# Function to calculate distinct and persistent machine ID
+get_machine_id() {
+    local machine_id
+
```

Comment:
Small suggestion, I would add some debug log traces here, just in case. At least one here, for instance

`log 0 "DEBUG" $NC "Trying to obtain Machine ID."`